compiler/plugins/lazy_interp.ml

```diff
@@ -84,9 +84,9 @@ let rec lazy_eval :
       let r, env1 = lazy_eval ctx env1 llevel e in
       if not (Expr.equal e r) then (
         log "@[<hv 2>{{%a =@ [%a]@ ==> [%a]}}@]" Print.var_debug v
-          (Print.expr ~debug:true ctx)
+          (Print.expr' ~debug:true ())
```

Comment:
Why `expr'` here? I find this `expr/expr'` quite confusing.

Reply:
They're the same function but with different arguments: the trouble is that you can't have automatic elimination of the optional `?debug` argument when you use the `Format.printf "%a" Print.expr e` form.

- The `expr` form skips the debug arguments, and so can be used directly in this form
- The `expr'` form has an added unit argument, allowing `Format.printf "%a" (Print.expr' ()) e`

I have seen this `f` / `f'` in a few projects, but indeed it's not very nice ; I'll try to find better naming.